CMakeLists.txt

```diff
@@ -60,6 +60,10 @@ if(APPLE)
         if (PLASMA_MAC_UNIVERSAL)
             set(CMAKE_OSX_ARCHITECTURES "$(ARCHS_STANDARD)")
         endif()
+
+        if(CMAKE_SYSTEM_VERSION VERSION_LESS 10)
```

Comment:
What is "10" here? Does `CMAKE_SYSTEM_VERSION` use the Darwin kernel version number (i. e. "10" is Mac OS X 10.6 Snow Leopard)?

Reply:
Yes, it's the Darwin version... maybe a comment is useful here